nifi-nar-bundles/nifi-slack-bundle/nifi-slack-processors/pom.xml

```diff
@@ -61,10 +61,6 @@
             <groupId>org.slf4j</groupId>
             <artifactId>jcl-over-slf4j</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.nifi</groupId>
-            <artifactId>nifi-api</artifactId>
-        </dependency>
```

Comment:
It seems like this should still be declared, perhaps the removal triggered the other addition with the compile scope, but this one looks correct.